src/services/API/sqs.js

```diff
@@ -10,6 +10,15 @@ const sqsClient = new SQSClient({
   },
 });
 
+/* This function is used to send an SQS message to the queue on AWS. Makes use of the AWS SDK. (SendMessageCommand)
+@params {String} - purpose
+@params {String} - purposeParams
+@params {String} - redirectTo
+@params {String} - redirectID
+@params {Array} - userIDList - list of users wanting to go through the layer
+@params {String} - authType
+Everything, except the userIDList, is extracted from the auth layer URL
+*/
```

Comment:
`@param` not `@params`
the format is:
`@param {type} {variable name} - {description}`
please remove the `-` between `{type}` and `{variable name}`.

+ format the docstring according to the feedback given before
+ add descriptions for each param
+ no need to mention implementation details like `SendMessageCommand`  in the docstring